gulpfile.js

```diff
@@ -253,6 +253,17 @@ function compile(watch, shouldMinify, opt_preventRemoveAndMakeDir,
       include3pDirectories: true,
       includePolyfills: false,
     }),
+    compileJs('./3p/', 'ampanalytics-lib.js',
+        './dist.3p/' + (shouldMinify ? internalRuntimeVersion : 'current'), {
+      minifiedName: 'ampanalytics-v0.js',
+      checkTypes: opt_checkTypes,
+      watch: watch,
+      minify: shouldMinify,
+      preventRemoveAndMakeDir: opt_preventRemoveAndMakeDir,
+      externs: ['ads/ads.extern.js',],
+      include3pDirectories: true,
+      includePolyfills: false,
+    }),
```

Comment:
Yep, putting it back in hoses things again. The block of code is:
`      closureCompile(['./3p/ampanalytics-lib.js'], './dist',
        'ampanalytics-check-types.js', {
          externs: ['ads/ads.extern.js'],
          include3pDirectories: true,
          includePolyfills: true,
          checkTypes: true,
        }),
`
The errors are stuff like this:
`
src/services.js:225: ERROR - Bad type annotation. Unknown type module$src$service$template_impl.Templates
  return /** @type {!./service/template-impl.Templates} */ (
                     ^ (the caret points to the dot when not using a variable-width font...)
`